app/models/emis_redis/military_information.rb

```diff
@@ -259,5 +261,28 @@ def service_history
         }
       end
     end
+
+    def guard_reserve_service_periods
+      @guard_reserve_service_periods ||= items_from_response('get_guard_reserve_service_periods')
+    end
+
+    def guard_reserve_service_by_date
+      @guard_reserve_service_by_date ||= lambda do
```

Comment:
is the lambda just to wrap the assignment? could it be wrapped in being ... end instead?